src/main/java/de/tum/in/www1/artemis/config/SecurityConfiguration.java

```diff
@@ -140,28 +141,28 @@ protected void configure(HttpSecurity http) throws Exception {
             .csrf()
             .disable()
             .addFilterBefore(corsFilter, UsernamePasswordAuthenticationFilter.class)
-                .exceptionHandling().authenticationEntryPoint(problemSupport)
-                .accessDeniedHandler(problemSupport)
-        .and()
+            .exceptionHandling().authenticationEntryPoint(problemSupport)
```

Comment:
Please revert the formatting changes in this file. The formatting was intentional to at least get some structure into the builder method call chains.